CHANGELOG.md

```diff
@@ -7,6 +7,12 @@
 Access of Google Cloud Storage buckets with Requester Pays enabled is now supported.
 Please read the [relevant documentation](http://cromwell.readthedocs.io/en/develop/filesystems/GoogleCloudStorage#requester-pays) for information on how to enable it and the consequences.
 
+### Updated MySQL client
```

Comment:
From a "why would a reader care" I'd unbury the "supports MySQL 8" lede into this header